### ISSUE TITLE
Clarify requirements around headphone levels

### ISSUE THREAD

**Issue by SmittyHalibut** [open-headset-interconnect-standard/ohis#21]
DrCampy: Hey, how are you doing?
I'm finalizing Module17.
I remember that we discussed how to implement OHIS on the board. 
I also realize that the current implementation is a notable deviation from the standard, so I'm not sure that it is a good idea to keep it as-is...

Most notably, the audio output we provide is the one that normally goes to the speaker. So the amplitude is up to 5V. But your document says that it is expected to be between 0dBV (1V) and -10dBV (300 mV). 

And if I want to stay close to 1V, I could repurpose one of the free operational amplifiers already on the board, but it is only able to provide a few mA (25 mA iirc) so it would not be able to power a load as low as 8 Ohms... . Also, it would be ground referenced but with a DC offset. If I want to cancel DC offset, I would need to put a DC-blocking cap but then, my low-pass cutoff frequency would be terrible if we must support up to 300 Ohms as stated in the standard

I need to know what you suggest for that. Also, maybe the standard needs to be more precise about what is acceptable (in terms of acceptable loads mostly).

Mark: Output: Just put a resistor in series, and that'll be fine.  I'm trying to figure out what value.  Probably something between 100R and 33R.  I know my stereo at home powers the headphones by just putting a 220R in series from the 100W speaker output!  If you do the math, it actually comes out pretty close.

DC blocking:  Yep, it's about 100uF or more.  And yes, it needs to be there.

I usually put 220uF or 470uF on stuff that I want to be full range.  For communications range, you can probably get away with 47uF (that's a guess, not actual calculation.)

And the low cut-off point goes DOWN (more bass) as your load impedance goes up.  It's not able to discharge the cap as quickly, so the cap is able to pass the lower frequencies better.

It's the 8 ohm side that you need to be worried about.

An 8 ohm load drawing through a 100uF cap has its -3dB point at 200Hz.  That's probably fine for this use case.

So, put a 100uF cap in series from the power amp, then two 33R resistors off the negative side of the cap toward the Left and Right channels on OHIS.  They'll serve dual purpose: Allowing one headphone to get shorted to ground without killing the other, and they'll brint a 5v (did you mean watt?) signal down to about right for headphones.

> put a 100uF cap in series...

No, it is indeed up to 5V signal (5V peak). Output power is 1W max

Thank you for the feedback, I ll crunch some numbers tomorrow and tell you what 😄

Also, I'm afraid that with serial resistors, I will have way more than 0dBV with higher impedance loads 😬

Mark: The good news is, high impedance loads can handle the voltage better and will actually need it.

DrCampy: Yes, but then maybe you should update the document with the standard to provide some guidance? And also include more details about the acceptable voltages wrt the impedances ?

*whew!* that's a lot of copy and paste.

**Comment by mdiepart** [comment]
I'll just add a few explanations about where things become hard to follow:
The standard does not specify impedance requirements. It states that speaker impedances can be between 8 and 64 Ohms but also that it can sometimes be as high as 300 Ohms, without specifying an upper limit. However, the standard does specify that the full volume amplitude must be between 0 dBV and -10 dBV. Thus an implementation would have to keep this amplitude at all impedances.


What is needed is to specify a range of impedances for which the device must respect specific amplitude levels. With an added note that any impedance outside of those ranges might work but with no guarantee.

The last problem is that, if using a resistor to do a divider bridge, then the full volume amplitudes won't be respected. Indeed, if we take the numbers specified in the current version of the specifications, there is a ratio of >37 between 8 ohms and 300 ohms, but a ratio slightly above 3. Thus, it is very difficult to follow the standard if the implementation is using a divider bridge (the driving voltage would need to be very large with a very very large impedance before the OHIS connector.
